### PR TITLE
Add GitHub Actions CI

### PR DESCRIPTION
**Pull request by wavefunction91**

### CODE REVIEW

File: .github/workflows/linalg/Makefile [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+	test -x /opt/intel/mkl/bin/mklvars.sh
+
+
+# BLIS
+download/blis:
+	mkdir -p ${software_prefix}/src/blis
+	cd ${software_prefix}/src/blis && wget --no-verbose https://github.com/flame/blis/archive/0.8.0.tar.gz
+	cd ${software_prefix}/src/blis && tar xzf 0.8.0.tar.gz
+
+install/blis/lp64-pthreads:
```
Comment (asadchev) [inline .github/workflows/linalg/Makefile:102]: Most of boiler code you can replace with patterns (%) and target variables

Comment (wavefunction91) [reply .github/workflows/linalg/Makefile:102]: I completely agree - I have not worked directly with Makefiles in quite some time, so I'm sure there's quite a bit of streamlining that can happen here. Would you be able to contribute these changes?

**Comment by asadchev** [comment]
Yes, need finish some valeev work, say by Tuesday?

**Comment by wavefunction91** [comment]
Sure, that works!

**Pull request merged**